### PR TITLE
fix #50908 i.ifeng.com

### LINKED ISSUE #50908: i.ifeng.com
### Issue URL (Ads)
 [http://i.ifeng.com](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=http%3A%2F%2Fi.ifeng.com)

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
Platform: | iOS
AdGuard version: | 2.1.2(173)
Browser: | Safari
System wide filtering: | disabled
Simplified filters: | disabled
AdGuard DNS: | None
Filters: | <b>Ad Blocking:</b><br/> AdGuard Base, <br/>AdGuard Mobile Ads<br/><br/><b>Privacy:</b><br/> AdGuard Tracking Protection<br/><br/><b>Language-specific:</b><br/> EasyList China

### PR DESCRIPTION
**Pull request by Sergey-Lyapin**
#50908

### CODE REVIEW

File: SocialFilter/sections/specific.txt [hunk 1]
```diff
@@ -43,6 +43,7 @@ hdfilmkamyonu.com,fullfilmvakti.com,indirmedenizleyin.com,hdfilmsizle1.com,hdfil
 pornopank.com,3w-video.com,coqporn.com,paraticlube.com,turkeytv.info,pornopix.net,turkceingilizceceviri.info,turkfirmalar.info,isexpro.net,javhdonline.com,pornoworlds.org##.dp-share
 !
 sadalskij.livejournal.com##.b-sharethis-livejournal
+ifeng.com##div[class^="share_box-"]
```
Comment (Alex-302) [inline SocialFilter/sections/specific.txt]: +

File: AnnoyancesFilter/sections/popups.txt [hunk 2]
```diff
@@ -89,6 +89,7 @@ sodapdf.com##.be-tooltip
 ! Specific sites
 !
 netfilmtvizle.com##body > #ics
+ifeng.com###root > a[class^="bookrack-"]
```
Comment (Alex-302) [inline AnnoyancesFilter/sections/popups.txt]: I don't see it. Please make screenshot

File: AnnoyancesFilter/sections/mobile-app.txt [hunk 3]
```diff
@@ -34,6 +34,11 @@ reddit.com##body > div[id="container"] > div[class="App"] > div[class="AppMainPa
 !
 mobile.donanimhaber.com##.mobil-forum-tanitim
 mobilkulup.com##.TapatalkBanner_Container
+ifeng.com###root > section[class^="fixSlide-"]
+ifeng.com###root > div[class^="bottom_box-"]
+ifeng.com###root > div[class^="container-"] > div[class^="bottomSlide-"]
+ifeng.com###root > div[class^="share-"]
```
Comment (Alex-302) [inline AnnoyancesFilter/sections/mobile-app.txt]: I don't see

File: AnnoyancesFilter/sections/mobile-app.txt [hunk 4]
```diff
@@ -34,6 +34,11 @@ reddit.com##body > div[id="container"] > div[class="App"] > div[class="AppMainPa
 !
 mobile.donanimhaber.com##.mobil-forum-tanitim
 mobilkulup.com##.TapatalkBanner_Container
+ifeng.com###root > section[class^="fixSlide-"]
+ifeng.com###root > div[class^="bottom_box-"]
+ifeng.com###root > div[class^="container-"] > div[class^="bottomSlide-"]
```
Comment (Alex-302) [inline AnnoyancesFilter/sections/mobile-app.txt]: don't see

File: AnnoyancesFilter/sections/mobile-app.txt [hunk 5]
```diff
@@ -34,6 +34,11 @@ reddit.com##body > div[id="container"] > div[class="App"] > div[class="AppMainPa
 !
 mobile.donanimhaber.com##.mobil-forum-tanitim
 mobilkulup.com##.TapatalkBanner_Container
+ifeng.com###root > section[class^="fixSlide-"]
+ifeng.com###root > div[class^="bottom_box-"]
```
Comment (Alex-302) [inline AnnoyancesFilter/sections/mobile-app.txt]: this is description for photos
image

File: AnnoyancesFilter/sections/mobile-app.txt [hunk 6]
```diff
@@ -34,6 +34,11 @@ reddit.com##body > div[id="container"] > div[class="App"] > div[class="AppMainPa
 !
 mobile.donanimhaber.com##.mobil-forum-tanitim
 mobilkulup.com##.TapatalkBanner_Container
+ifeng.com###root > section[class^="fixSlide-"]
```
Comment (Alex-302) [inline AnnoyancesFilter/sections/mobile-app.txt]: works as mobile app download button. Can be blocked

File: AnnoyancesFilter/sections/mobile-app.txt (hunk 5 above)
Comment (Sergey-Lyapin) [reply AnnoyancesFilter/sections/mobile-app.txt]: but it doesn't lays in `#root` directly

File: AnnoyancesFilter/sections/mobile-app.txt (hunk 4 above)
Comment (Sergey-Lyapin) [reply AnnoyancesFilter/sections/mobile-app.txt]: <details>
<summary>Screenshot </summary>

</details>

File: AnnoyancesFilter/sections/mobile-app.txt (hunk 5 above)
Comment (Sergey-Lyapin) [reply AnnoyancesFilter/sections/mobile-app.txt]: <details>
<summary>Screenshot </summary>

</details>

File: AnnoyancesFilter/sections/mobile-app.txt (hunk 3 above)
Comment (Sergey-Lyapin) [reply AnnoyancesFilter/sections/mobile-app.txt]: <details>
<summary>Screenshot </summary>

</details>

File: AnnoyancesFilter/sections/popups.txt (hunk 2 above)
Comment (Sergey-Lyapin) [reply AnnoyancesFilter/sections/popups.txt]: <details>
<summary>Screenshot </summary>

</details>

**Review by Alex-302 — APPROVED** [review]

**Pull request merged**